Packs/FidelisEndpoint/Integrations/FidelisEndpoint/FidelisEndpoint.py

```diff
@@ -0,0 +1,1577 @@
+                    "filters": [
+                        {
+                            "filterType": "criteria",
+                            "column": "dnsQuestion",
+                            "operator": "=~",
+                            "value": domain
+                        },
+                        {
+                            "filterType": "criteria",
+                            "column": "localIP",
+                            "operator": '=',
+                            "value": source_ip
+                        }
+                    ]
+                }
+            }
+        }
+        response = self._http_request('POST', url_suffix, json_data=body)
+        if response.get('error'):
+            return_error(response.get('error'))
+        if 'data' in response:
+            return response.get('data')
+        return {}
+
+    def query_events(self, start_time: str, end_time: str, logic: str, column: str, value: str, entity_type: str):
+
+        url_suffix = '/v2/events'
+        body = {
+            "dateRange": {
+                "start": start_time,
+                "end": end_time
+            },
+            "resultFields":
+                ["endpointName", "eventType", "processStartTime", "parentName", "pid", "name", "path", "user", "hash",
+                 " parameters"],
+
+            "criteriaV3": {
+                "relationshipFilter": None,
+                "entityType": entity_type,
+                "filter": {
+                    "filterType": "composite",
+                    "logic": logic,
+                    "filters": [
+                        {
+                            "filterType": "criteria",
+                            "column": column,
+                            "operator": '=',
+                            "value": value
+                        }
+                    ]
+                }
+            }
+        }
+
+        response = self._http_request('POST', url_suffix, json_data=body)
+        if response.get('error'):
+            return_error(response.get('error'))
+        if 'data' in response:
+            return response.get('data')
+        return {}
+
+
+def test_module(client: Client, *_):
+    """
+    Returning 'ok' indicates that the integration works like it is supposed to. Connection to the service is successful.
+
+    Args:
+        client: HelloWorld client
+
+    Returns:
+        'ok' if test passed, anything else will fail the test.
+    """
+
+    client.test_module_request()
+    demisto.results('ok')
+    return '', {}, {}
+
+
+def list_alerts_command(client: Client, args: dict):
+    """get information about alerts. """
+
+    skip = args.get('skip', 0)
+    limit = args.get('limit')
+    sort = args.get('sort')
+    facet_search = args.get('facet_search')
+    start_date = args.get('start_date')
+    end_date = args.get('end_date')
+    headers = ['Name', 'EndpointName', 'EndpointID', 'Source', 'ArtifactName', 'IntelName', 'Severity', 'ID',
```

Comment:
make sure `ID` is the first header in all commands outputs
add `CreateDate` to headers